### ISSUE TITLE
Parallel Call to external with ACD

### ISSUE THREAD

**Issue by getsolveit** [amooma/GS5#76]
It was helpful to select more than only sip accounts in agents in ACD.

**Comment by jpawlowski** [comment]
Did you try to setup a SIP account with permanent call-forwarding to external and register this SIP account with the ACD?

**Comment by getsolveit** [comment]
No, I don't try it. I have on my fon a call-forwarding, but the other agents stop ringing now. I test it with permanent call-forwarding 3 month ago and it don't work.

**Comment by jpawlowski** [comment]
3 month ago sounds more you were trying it with GS3, not GS5. I think it's worth trying, otherwise please ask on the mailing list first before opening a ticket if you are not absolutely sure the feature is really missing or not possible with GS5.

**Comment by getsolveit** [comment]
No, I have get a GS5 Installation from Stefan. And i have done all updates with git pull.

**Comment by wintermeyer** [comment]
If you pay for it we can include it. But until than I have to say that we have bigger fish to fry feature request wise.

**Comment by wintermeyer** [comment]
@spag How much time would it take to implement such a feature? Ballpark.

**Comment by spag** [comment]
Implementing this would require some serious rewiring in the ACD module - both dialplan and GUI wise.
